src/Ombi.Mapping/Ombi.Mapping.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="AutoMapper" Version="6.1.1" />
     <PackageReference Include="AutoMapper.Collection" Version="3.1.2" />
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="3.0.1" />
+    <PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="1.1.9" />
```

Comment:
Why has this been added?

Reply:
While running the ARM build I ran into an issue similar to this: https://github.com/ericsink/SQLitePCL.raw/issues/161

```Unhandled Exception: System.TypeInitializationException: The type initializer for 'Microsoft.Data.Sqlite.SqliteConnection' threw an exception. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.DllNotFoundException: Unable to load DLL 'e_sqlite3': The specified module could not be found.```

Adding this solved it. I haven't dabbled in .NET in a long time, so if there's a better way to solve that issue...

Reply:
sorry for the long reply, this is a strange project to include this reference, you 100% sure you meant for it to be in this project and not Ombi.Store?

Reply:
I'll try that, traveling right now.